src/api.ts

```diff
@@ -189,25 +189,28 @@ export default class Chromeless<T extends any> implements Promise<T> {
   /**
    * Get the cookies for the current url
    */
```

Comment:
I'm having a hard time understanding how this all works here (and even before). @timsuchanek or @adieuadieu I'd love to hear your feedback on this